jfix-zookeeper/src/main/java/ru/fix/zookeeper/transactional/TransactionalClient.java

```diff
@@ -78,6 +78,21 @@ public TransactionalClient createPathWithParentsIfNeeded(String path) throws Exc
         return this;
     }
 
+    /**
+     * Usage: invoke this method with the same parameter during different transactions,
+     * which must not succeed in parallel.
+     *
```

Comment:
better to add notes to javadoc: 

node by given path should exist, value of given node will be overwritten

Reply:
added

Reply:
Actually, this javadoc doesn't explain what the method is doing. 
"invoke this method with the same parameter during different transactions" - does it mean, it should always be used with the same parameters? 
As I see, it should be something like "Adds an updating version operation at the given path. Update version - operation, that <...> at the given path <...>. Path should exists, <what happens> elsewhere."